### DISCUSSION TITLE
Play (pre)loaded bytes directly (e.g. connecttoMem)

### DISCUSSION THREAD

**Discussion by SndrSchnklshk** [schreibfaul1/ESP32-audioI2S#330] [Ideas]
First of all; great lib and nice work! I'm using and it works perfectly fine:
`connecttoFS(fs::FS &fs, const char* path, uint32_t resumeFilePos)`

I was wondering, how can I easily replay (small files) sounds without (re)loading it from the FS? Please tell me if it's out of scope for your project, but it would be a nice addition to me.

Can I access the loaded bytes and replay them directly when needed? With a calls like:
Access the loaded bytes:
 `uint16_t* getLoadedBytes()` 
Play directly:
 `void connecttoMem(uint16_t* bytes, uint32_t len, unsigned  uint32_t resumeFilePos)`
 
Thanks!

**Comment by schreibfaul1** [comment]
Well, that's not easy. In the case of compressed audio formats, it should then be ensured that the memory jumps to the beginning of a frame if possible.

**Reply by SndrSchnklshk** [reply]
Yes, I can see the difficulty. What about for uncompressed formats? 

After a song has been finished, currently, the memory will cleaned-up right? How to replay without reloading it from FS again?
